jscomp/runtime/caml_obj.ml

```diff
@@ -181,8 +181,13 @@ let rec caml_compare (a : Obj.t) (b : Obj.t) : int =
     match is_a_number , is_b_number with 
     | true, true -> 
       Pervasives.compare (Obj.magic a : int) (Obj.magic b : int)
-    | true , false -> -1 (* Integer < Block in OCaml runtime GPR #1195 *)
-    | false, true -> 1 
+    | true , false -> 
```

Comment:
Line 172 above (sorry don't know how to put the comment there) is the first that should be changed if we adopt this order: https://gist.github.com/cristianoc/791aac26f94dbded0fc137d61f4bd2a8